src/bindgen/ir/constant.rs

```diff
@@ -23,6 +23,10 @@ use syn::UnOp;
 #[derive(Debug, Clone)]
 pub enum Literal {
     Expr(String),
+    PostfixUnaryOp {
+        op: &'static str,
+        value: Box<Literal>,
```

Comment:
Could you elaborate on the reasoning behind this change? it seems this could be a separate commit.

Reply:
Replacing `fmt::Display` implements with implementing `write` method occur error to make `Expr` Literal by stringifying unary expression. So, I introduced a new `PostfixUnqaryOp`.
I thought it is following changes by implementing the `write` method. Is it better making a separate commit?